app/src/hooks/useBioHubApi.ts

```diff
@@ -117,13 +123,39 @@ export const useBiohubApi = () => {
     return data;
   };
 
+  /**
+   * Fetch the media list.
+   *
+   * @return {*} {Promise<IMedia[]>}
+   */
+
+  const getMediaList = async (projectId: string): Promise<IMedia[]> => {
+    //const api = await apiPromise;
+
+    const { data } = await api.get('/api/projects/' + projectId + '/artifacts/list');
+
+    const mediaKeyList: IMedia[] = [];
+
+    if (data && data.length > 0) {
```

Comment:
This is also somewhat subjective, but I find it makes for cleaner code if instead of wrapping the happy path in an if statement, we wrap the unhappy path instead, and fail fast.  So in this example:

```Typescript
const mediaKeyList: IMedia[] = [];

if(!data || !data.length) {
  return mediaKeyList;
}

data.forEach((file: any) => {...});

return mediaKeyList;
```
If data is null or has no length, we return immediately.  It also removes 1 level of nesting from the 'real' forEach logic below, which makes it a bit easier to read.